humanmine.ipynb

```diff
@@ -331,206 +341,390 @@
-      ],
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "d06de2ae91584a9e98eba40dd39504f7",
+       "version_major": 2,
+       "version_minor": 0
+      },
       "text/plain": [
-       "<IPython.core.display.HTML object>"
+       "AppLayout(children=(Dropdown(description='Disease: ', layout=Layout(grid_area='header'), options=('Adenocarcin…"
       ]
      },
      "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "html = \"<table>\"\n",
-    "for k, v in images.items():\n",
-    "    html += '<tr><td><h4>Developmental stage: '+k+'</h4></td></tr><tr>'\n",
-    "    for i in v:\n",
+    "from ipywidgets import AppLayout, widgets\n",
+    "\n",
+    "increase = 5\n",
+    "max_value = increase\n",
+    "min_value = 0\n",
+    "\n",
+    "disease = \"\"\n",
+    "\n",
+    "for k, v in images:\n",
+    "    if k != PATHOLOGY_NORMAL_VALUE:\n",
+    "        disease_list.append(k)\n",
+    "\n",
+    "def display_images(images, min, max):\n",
+    "    html = \"<table>\"\n",
+    "    html += '<tr>'\n",
+    "    if min < 0:\n",
+    "        min = 0\n",
+    "    if max >= len(images):\n",
+    "        max = len(images)\n",
+    "\n",
+    "    \n",
+    "    for i in images[min:max]:\n",
     "        qs = {'id': i}\n",
     "        url = IMAGE_DATA_URL.format(**qs)\n",
     "        url_link = LINK_URL.format(**qs)\n",
-    "        html += '<td><a href=\"'+url_link+'\" target=\"_blank\"><img src=\"'+url+'\"/></a></td>'\n",
+    "        html += '<td><a href=\"'+url_link+'\" target=\"_blank\"><img src=\"'+url+'\"/></a>&nbsp;</td>'\n",
     "    html += \"</tr>\"\n",
-    "html += \"</table>\"\n",
-    "display(HTML(html))"
+    "    html += \"</table>\"\n",
+    "    html_widget.value = html\n",
+    "    \n",
+    "def on_selection_change(change):\n",
+    "    global disease\n",
+    "    if change['name'] == 'value':\n",
+    "        values = get_images(change['new']) \n",
+    "        if values is None:\n",
+    "            return\n",
+    "        disease = change['new']\n",
+    "        min_value = 0\n",
+    "        max_value = increase \n",
+    "        display_images(values, 0, max_value)\n",
+    "                \n",
+    "def get_images(disease):\n",
```

Comment:
Could we please have the number of images available form each disease displayed immediately once I select the disease in the dropdown ? This would improve the experience and give the user a hand when he/she gets lost due to the "Next" and "Previous" buttons weirdness, see https://github.com/IDR/idr-notebooks/pull/116/files#r903566397